### PR TITLE
Improve README

### PR DESCRIPTION
**Pull request by pereorga**

### CODE REVIEW

File: README [hunk 1]
```diff
@@ -40,7 +40,7 @@ Requirements:
 
 Optional dependencies:
 
-  * cld2 (better language autodetection and non-English source languages)
```
Comment (vslavik) [inline README]: This change is semantically incorrect — it really is *and*, not *in*.

File: README [hunk 2]
```diff
@@ -53,7 +53,7 @@ Do the usual thing:
     make
     make install
 
-You must have the dependencies installed in a location configure will find,
```
Comment (vslavik) [inline README]: I think "will" is better here, and a better fix would be "in a location *where* configure will find _them_,"

File: README (hunk 1 above)
Comment (pereorga) [reply README]: Ok, I was unsure about this one (did not understand the meaning). I will update

File: README (hunk 2 above)
Comment (pereorga) [reply README]: I see, your suggestion makes sense (it's the "dependencies", not the "location", that `configure` will find). I will update

**Comment by pereorga** [comment]
Pushed. Also, removed a trailing space.

File: README [hunk 3]
```diff
@@ -12,15 +12,15 @@
...
+with translating applications into another language. For details on principles
 of the solution used, see GNU gettext documentation or wxWidgets' wxLocale
 class reference.
 
 
  Installing
 ============
 
-Easily-installable prebuilt binaries are provided for Windows and macOS and
+Easily-installable prebuilt binaries are provided for Windows and macOS are
```
Comment (c72578) [inline README]: The second "are" in the sentence is not correct yet.
Easily-installable prebuilt binaries are provided for Windows and macOS **are** available from https://poedit.net/download.

How about a shortened version:
Easily-installable prebuilt binaries for Windows and macOS are available from https://poedit.net/download.

Comment (pereorga) [reply README]: Right, that makes sense

**Comment by vslavik** [comment]
Thanks!

**Pull request merged**